### PR TITLE
[WIP] use upstream janusgraph image

### PR DESCRIPTION
**Pull request by idcrosby**

### CODE REVIEW

**Comment by idcrosby** [comment]
This doesn't (yet) use the `run.sh` script to check dependencies. If needed we can find a way to add this.

File: docker-compose-dev.yml [hunk 1]
```diff
@@ -53,19 +53,16 @@ services:
...
     restart: on-failure:0
   janus:
-    image: janusgraph:0.2.0-hadoop2
-    build:
-      dockerfile: Dockerfile.janus
-      args:
-        version: 0.2.0
-        hadoop: hadoop2
-      context: ./tools/dev/janusgraph-docker/
+    image: gcr.io/cloud-solutions-images/janusgraph:v2
```
Comment (moretea) [inline docker-compose-dev.yml]: This version is unacceptable. We need at least version v3.0.0, ideally the latest one v3.0.1

Comment (idcrosby) [reply docker-compose-dev.yml]: Ok, in that case it does look like we will need to build/maintain our own Janusgraph image. I'll close this and start work on setting that up.

Comment (idcrosby) [reply docker-compose-dev.yml]: Also, fyi, looks like there is a plan to create an official janusgraph docker image... https://github.com/JanusGraph/janusgraph-docker/pull/1

**Pull request closed without merging**